src/assets/locales/resources_en_US.json

```diff
@@ -1,4 +1,8 @@
 {
+  "alert_close": {
```

Comment:
Should these be prefixed with `skyux_indicators_` ?

Reply:
Good catch!